### DISCUSSION TITLE
RG351P: Cut power to internal wifi module when wifi is disabled in menu

### DISCUSSION THREAD

**Discussion by nfriedly** [351ELEC/351ELEC#653] [Known Issues & Solutions]
### Is your feature request related to a problem? Please describe.
For an RG351P with an internal WiFi module, there's no way to completely disable it in 351Elec. Because of this, it constantly draws power (~0.5W) and emits heat, even when wifi is disabled.

### Describe the solution you'd like
I'd like for the Wi-Fi toggle in the Emulation Station Networking menu to cut power to the internal WiFi module when toggled off. This is the behavior of the stock firmware.

(There is supposed to be an additional pin to tell the module to disable it's radio, but it doesn't seem to be connected to anything, which is probably why the stock FW cuts the power instead.)

### Describe alternatives you've considered
Dremeling a hole into the side of my unit and adding a physical switch to disable power to the internal Wifi module

### Additional context
Here's a photo I made for reference:
![PXL_20210924_173923025(1)](https://user-images.githubusercontent.com/114976/134730134-545b32d6-8d4e-4095-a98c-623cd352e701.jpg)

**Comment by dhwz** [comment]
this is already done in latest pre-release images

**Reply by nfriedly** [reply]
Oh, awesome! I ended up selling my 351P, but this is good news for other folks :)
